index.html

```diff
@@ -4,26 +4,27 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We" crossorigin="anonymous">
     <link rel="stylesheet" href="./css/main.css" />
     <link rel="icon" href="./favicon.ico" />
     <title>Aziz Mejri - Portfolio</title>
   </head>
   <body>
     <header>
-      <h1 id="logo">Aziz Mejri</h1>
-      <button type="button" class="menu-bars">
-        <i class="fas fa-bars"></i>
-      </button>
-      <nav id="desktop-nav">
-        <ul>
-          <li><a class="nav-link" href="#projects">Portoflio</a></li>
-          <li><a class="nav-link" href="#about-me">About</a></li>
-          <li>
-            <a class="nav-link" href="#contact"
-              >Contact <i class="far fa-envelope"></i
-            ></a>
-          </li>
-        </ul>
+      <nav class="navbar navbar-dark d-flex justify-content-around">
+        <div class="container-fluid">
+          <h1 class="logo navbar-brand text-light">Aziz Mejri</h1>
+          <button class="navbar-toggler menu-bars" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
```

Comment:
Seems like you missed this comment here, from the previous review.  The nav bar is still not yet functional.  see the screenshot for the requirements still missing.

<img width="767" alt="Screen Shot 2021-09-05 at 8 20 24 pm" src="https://user-images.githubusercontent.com/70282171/132137329-b26d4fba-9f83-4772-ae8c-de3c7a31474b.png">

To get this features up and running, we need to add some bit of javascript to open and close the mobile menu. Here is a  good example and [tutorial from W3schools](https://www.w3schools.com/howto/howto_js_mobile_navbar.asp). Hope this helps